README.adoc

```diff
@@ -17,26 +17,42 @@ All industry use cases will be kept up to date with the latest Lisk SDK release.
 Also, you'll find an `/archive` folder that contains older examples referring to previous Lisk SDK releases.
 At the moment, the `/archive` folder contains examples for Lisk SDK `v2`, `v3` and `v4`.
```

Comment:
The `archive` folder has been removed, so please remove this statements as well.